app/controllers/groups_controller.rb

```diff
@@ -2,6 +2,29 @@ class GroupsController < ApplicationController
 
   include GroupsHelper
 
+  skip_before_filter  :verify_authenticity_token
```

Comment:
All of these functions are used for get/post requests from rails when a new chat or benny reply needs to be stored/retrieved from the redis keystore, or the history needs to be cleared. groupname:groupid:chathistory is the key for a list holding chat messages, and groupname:groupid:bennyhistory is the key for a list holding benny json replies from the server.